### ISSUE TITLE
Autocomplete onboarding does not appear on macOS Ventura

### ISSUE THREAD

**Issue by Nah-Nova** [withfig/fig#1986]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.0.0 (22A380)

### Expected behaviour

A working version of autocomplete, the only time autocomplete works is in the introduction of autocomplete 

### Actual behaviour

When enabling autocomplete in the dashboard this problem occurs
<img width="318" alt="Screenshot 2022-11-04 at 12 20 12" src="https://user-images.githubusercontent.com/56248103/199961339-25620643-654e-4c1a-9310-490ddc6a138f.png">

### Steps to reproduce

brew install fig, then follow the steps in fig. When 
<img width="184" alt="Screenshot 2022-11-04 at 12 24 20" src="https://user-images.githubusercontent.com/56248103/199961442-3adadaba-3ca3-4ae6-9428-193521083674.png">
is pressed this error occurs
<img width="318" alt="Screenshot 2022-11-04 at 12 20 12" src="https://user-images.githubusercontent.com/56248103/199961477-f40b4703-764c-47f0-bfc9-d54b4795d448.png">

### Environment

```yaml
fig-details:
  - 2.7.4
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 8
  - mem: 32.00 GB
os-info:
  - macOS 13.0.0 (22A380)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/noaheutz/Developer
  - exe-path: /opt/homebrew/bin/fig
  - install-method: brew
  - env-vars:
    - FIGTERM_SESSION_ID: d444605f-7e2c-4155-b03c-e4abff548d9c
    - FIG_HOSTNAME: Noas-MacBook-Pro.local
    - FIG_TERM: 2.7.4
    - PATH: /Users/noaheutz/.serverless/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Library/Apple/usr/bin:/Users/noaheutz/.fig/bin:/Users/noaheutz/.local/bin:/Users/noaheutz/Library/Android/sdk/emulator:/Users/noaheutz/Library/Android/sdk/tools:/Users/noaheutz/Library/Android/sdk/tools/bin:/Users/noaheutz/Library/Android/sdk/platform-tools“alias
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - FIG_PID: 15373
    - FIG_PARENT: d444605f-7e2c-4155-b03c-e4abff548d9c
    - FIG_SET_PARENT: d444605f-7e2c-4155-b03c-e4abff548d9c
```

**Comment by rwhite27** [comment]
Having the same issue 😢

**Comment by Nah-Nova** [comment]
got it to work sort of, when I type cd it does not autocomplete
![Screenshot 2022-11-07 at 13 53 37](https://user-images.githubusercontent.com/56248103/200315333-e106c035-05c3-4f5c-847c-21669413521b.png)

but when I type yr it does
![Screenshot 2022-11-07 at 13 54 46](https://user-images.githubusercontent.com/56248103/200315567-dee53d30-344c-4f5d-a356-09289e3e4d60.png)

**Comment by Nah-Nova** [comment]
issue has been fixed, the problem was my local ohmyzsh install

**Comment by grant0417** [comment]
This was fixed on 2.7.5

**Comment by andrewchandra-tiket** [comment]
@Nah-Nova @grant0417 What's the solution, it also occurs on my MacOS Ventura
I use fig_cli 2.17.0
